index.html

```diff
@@ -215,6 +215,42 @@ <h5 class="text-white">Skills</h5>
               </ul>
             </div>
         </section>
+      <form action="https://formspree.io/f/xlezlkyw" method="post">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <input name="full-name" type="text" placeholder="Full name" maxlength="30" required>
+        <input name="email" type="email" placeholder="Email address" required>
+        <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <button class="get" type="submit">Get in touch</button>
+        <hr class="line">
 
+        <ul class="icons">
+          <li>
+            <a href="#" class="social-media-icons">
+              <img src="Images/ico/git.png" alt="Github icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icons">
+              <img src="Images/ico/linked.png" alt="Linkedin icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icons">
+              <img src="Images/ico/ange.png" alt="Angel icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icons">
+              <img src="Images/ico/twit.png" alt="Twitter icon for check"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icons">
+              <img src="Images/ico/med.png" alt="medium icon for check"/>
+            </a>
+          </li>
+          </ul>
+          <img class="shape" src="Images/ico/Shape.png" alt="shape">
```

Comment:
- [ ] For semantics purposes Please remove these social links list from the `<form>` tag and add it to a `<footer>` tag.